cfme/tests/services/test_orchestration_template.py

```diff
@@ -2,7 +2,6 @@
 import fauxfactory
 import pytest
 
-from cfme.cloud.provider import CloudProvider
```

Comment:
this module doesn't depend on any provider except vCloud. which we don't have atm